src/simple_cache_client.rs

```diff
@@ -103,7 +103,7 @@ impl SimpleCacheClientBuilder {
     }
 
     pub fn build(self) -> SimpleCacheClient {
-        let agent_value = format!("rust:{}", VERSION).to_owned();
+        let agent_value = format!("rust:{}", VERSION);
```

Comment:
should we make this `rust-cli` maybe? just wondering if we want be able break out CLI use vs someone using rust sdk for real in a project?

Reply:
I think we are already on the same page about this but I was thinking to use this new function [here](https://github.com/momentohq/client-sdk-rust/pull/39/files#diff-03c0f9db265c085414e8facf02f2c1b0cf681cb91221b1170970cf91886f0f41R212) in Momento CLI so that when we initialize a client in the CLI, [this](https://github.com/momentohq/client-sdk-rust/pull/39/files#diff-03c0f9db265c085414e8facf02f2c1b0cf681cb91221b1170970cf91886f0f41R217) `agent_value` gets used instead of `rust:<VERSION>`.